doc/src/user_guide.rst

```diff
@@ -93,6 +96,39 @@ To manage installation of Python dependencies we strongly recommend
 using `pip <https://pypi.python.org/pypi/pip>`_ and
 `virtualenv <https://pypi.python.org/pypi/virtualenv>`_.
 
+Quick-start Mac OS X
+^^^^^^^^^^^^^^^^^^^^
+
+To quickly install PyFR on a Mac we recommend using the package manager
+`homebrew <https://brew.sh/>`_.
+Open the terminal and install the dependencies with the following commands::
+
+    brew install python3 cmake open-mpi openblas metis
+    brew cask install paraview
+    pip3 install virtualenv
+
+Then create the virtual environment and activate it::
+
+    virtualenv --python=python3 ENV3
+    source ENV3/bin/activate
+
+Finally install PyFR with `pip <https://pypi.python.org/pypi/pip>`_ in the
+current virtual environment::
+
+    pip install pyfr
+
+This concludes the installation. In order to run PyFR with the OpenMP backend
+(see `Running PyFR`_),
+use the following settings in the configuration file (``.ini``)::
+
+    [backend-openmp]
+    cc = gcc-8
+    cblas = /usr/local/opt/openblas/lib/libblas.dylib
```

Comment:
Mac's have traditionally come with a BLAS library called Accelerate.  Is this no longer the case?

Reply:
So I suspect we might need libcblas.dylib rather than libblas.dylib.  Also, do we know if this is the same BLAS that is part of the Accelerate framework?

Reply:
libcblas.dylib and libblas.dylib are links to the same file in the Accelerate framework.